lib/pf/vlan.pm

```diff
@@ -495,6 +496,8 @@ sub getNormalVlan {
         $logger->info("[$mac] Username was NOT defined or unable to match a role - returning node based role '$role'");
     }
     $vlan = $switch->getVlanByName($role);
+    my $vlanpool = new pf::vlan::pool;
+    $vlan = $vlanpool->getVlanPool($vlan, $switch, $ifIndex, $mac, $node_info, $connection_type, $user_name, $ssid, $radius_request, $role);
```

Comment:
my $pool_args = {vlan => $vlan, switch => $switch, ifIndex => $ifIndex, mac => $mac, node_info => $node_info, connection_type => $connection_type, user_name => $user_name, ssid => $ssid, radius_request => $radius_request, role => $role};
$vlan = $vlanpool->getVlanPool($pool_args);

It makes it easier to add new parameters later